assets/js/components/event/ParticipantsList.tsx

```diff
@@ -61,26 +95,36 @@ const Participant = ({ client, participant, eventChannel }: ParticipantProps) =>
     ? "Moderator"
     : participant.isPresenter
     ? "Presenter"
-    : "Praticipant";
+    : "Participant";
+
+  const isClientPariticipant: Boolean = client.email == participant.email;
 
   return (
     <div className="Participant">
       <Tooltip
-        label={`${role}${client.name == participant.name ? " (You)" : ""}`}
+        label={`${role}${isClientPariticipant ? " (You)" : ""}`}
         borderRadius="25px"
         fontSize={"1.3rem"}
         className="InfoTooltip"
       >
         {icon}
       </Tooltip>
       <p className="ParticipantText">{participant.name}</p>
+      {participant.requestPresenting && (
+        <Tooltip borderRadius="25px" fontSize={"1.3rem"} className="InfoTooltip">
+          <QuestionCircle className="ParticipantIcon" />
+        </Tooltip>
+      )}
```

Comment:
I think it would be nice to add `label` to the Tooltip with something like "This user is asking to become presenter". Now I'm not sure what does the "?" icon means.

Reply:
I can't find in raising hand icon, so I decided to go with question mark. This way user can signal that he has some questions and want to say it publicly.

Reply:
I don't say that the icon is bad, but that you should add `label` prop to the `Tooltip` with some meaningful message, what's the point of using `Tooltip` without the `label`?